### PR TITLE
Update view when layout setting changes

### PR DESCRIPTION
**Pull request by artonge**
[Screencast from 2023-06-06 17-34-44.webm](https://github.com/nextcloud/photos/assets/6653109/822fb253-3812-4851-96d7-367640e3a2c9)

### CODE REVIEW

File: src/components/File.vue [hunk 1]
```diff
@@ -98,7 +97,6 @@ export default {
 		Star,
 		VideoIcon,
 	},
-	mixins: [UserConfig],
```
Comment (artonge) [inline src/components/File.vue:101]: Seems unused.

**Review by Pytal — APPROVED** [review]

**Pull request merged**